src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -1,19 +1,168 @@
 package frc.robot.subsystems;
 
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+import static frc.robot.Constants.Intake.*;
 
+/**
+ * Represents the robot's dual intakes.
+ * 
+ * <p>
+ * Each intake has a double solenoid connected to two pistons and Talon 
+ * SRX connected to a BAG motor.
+ */
 public class IntakeSubsystem extends SubsystemBase {
-    public void IntakeSubsystem() {
 
+    private DoubleSolenoid leftIntakeSolenoid;
+    private DoubleSolenoid rightIntakeSolenoid;
+
+    private WPI_TalonSRX leftIntakeMotor;
+    private WPI_TalonSRX rightIntakeMotor;
+    
+    public IntakeSubsystem() {
+        leftIntakeSolenoid = new DoubleSolenoid(
+            Constants.PCM_CAN_ID,
+            Constants.PCM_TYPE,
+            LEFT_INTAKE_EXTENSION_SOLENOID_ID,
+            LEFT_INTAKE_RETRACTION_SOLENOID_ID
+        );
+
+        rightIntakeSolenoid = new DoubleSolenoid(
+            Constants.PCM_CAN_ID,
+            Constants.PCM_TYPE,
+            RIGHT_INTAKE_EXTENSION_SOLENOID_ID,
+            RIGHT_INTAKE_RETRACTION_SOLENOID_ID
+        );
+
+        leftIntakeMotor = new WPI_TalonSRX(LEFT_INTAKE_MOTOR_ID);
+        rightIntakeMotor = new WPI_TalonSRX(RIGHT_INTAKE_MOTOR_ID);
     }
 
-    @Override
-    public void periodic() {
-        // This method will be called once per scheduler run
+    /**
+     * Extend the left intake without turning on the rollers
+     */
+    public void extendLeftIntake () {
+        leftIntakeSolenoid.set(Value.kForward);
+    }
+
+    /**
+     * Extend and activate the left intake
+     */
+    public void deployLeftIntake () {
```

Comment:
"Extend" and "Deploy" are a bit ambiguous. I would expect them (by name alone) to be doing the same thing.
Naming is a very important, and often challenging thing to get right. It's important because when you're deep in code using these APIs, you don't want any confusion. Yes, there's the comment, but it's best to start with very clear an unambiguous function names.

To help prevent confusion, I'd recommend renaming one of them to something more clear.

If the robot will never start the intake motor without being extended, you might consider changing "Deploy" to "ExtendAndActivate". There's nothing wrong with being verbose in your naming. In fact, since deploy is exactly that, you could also just combine both functions into a single function called "deploy" that takes a boolean to start the motor or not.